assets/onboarding/data/selectors.js

```diff
@@ -34,12 +34,15 @@ export const isSubmitting = ( state ) => state.isSubmitting;
  */
 export const getSubmitError = ( state ) => state.submitError;
 
+/* eslint-disable jsdoc/check-param-names */
 /**
  * Step state selector.
  *
- * @param {Object} state Current state.
- * @param {string} step Step name.
+ * @param {Object}  state         Current state.
+ * @param {string}  step          Step name.
+ * @param {boolean} shouldResolve Flag whether should invoke the resolver.
```

Comment:
It was documented here to turn explicit that it can receive this parameter, because the resolver is tied with the selector, receiving the same parameters.